.circleci/config.yml

```diff
@@ -390,13 +385,40 @@ commands:
             git status
             gh pr create --fill --base main --head $CIRCLE_BRANCH
 
+  build-and-push-docker-image:
```

Comment:
This step name should be more descriptive. What docker image are we building and pushing?

Reply:
Renamed the job to this.
build-and-push-custom-node-image